src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -14,6 +14,7 @@
     public static final String MESSAGE_INVALID_SERIAL_NUMBER = "Invalid Serial Number! \n%1$s";
 
     public static final String MESSAGE_DUPLICATE_BORROWER = "Phone/Email is already in used! ";
+    public static final String MESSAGE_NO_SUCH_BORROWERID = "No such borrower ID!";
```

Comment:
I think there should be an underscore between "BORROWER" and "ID", i.e., it should be MESSAGE_NO_SUCH_BORROWER_ID

Reply:
I agree! Will make the necessary amendments !